### DISCUSSION TITLE
Add challenges with friends

### DISCUSSION THREAD

**Discussion by AbdelrahmanDwedar** [codewars/codewars.com#3012] [Ideas]
My idea is to add section where you and some friend can start challenge together and when someone solve the problem first that person wins.

**Comment by jroycodes** [comment]
> My idea is to add section where you and some friend can start challenge together and when someone solve the problem first that person wins.

That would be a great idea.

**Comment by Perry-Bright** [comment]
yeah that will be very great, like a multiplayer challenge with limited time and one target where players can choose the language they wish to use

**Reply by AbdelrahmanDwedar** [reply]
Yes exactly what I was thinking about.

**Reply by max021x** [reply]
i think it must have a timer to like 10 min or more  😍

**Reply by AbdelrahmanDwedar** [reply]
> i think it must have a timer to like 10 min or more  😍
> 

Amazing idea, maybe also make it that we can set a time manually

**Comment by Madjosz** [comment]
Codewars is not focussed on speed, neither in solving time nor in execution time. Given that and also the huge backlog combined with the low manpower it is unlikely that such a feature will be implemented anytime soon.

In the meantime I could refer y'all to [CodinGame](https://www.codingame.com/servlet/urlinvite?u=4814675) which provides exactly what you are looking for with "CLASH OF CODE".

**Reply by Kacarott** [reply]
For a focus on golfing, I'd suggest [code.golf](https://code.golf/)
